### PR TITLE
chore: format files with clang

### PR DESCRIPTION
**Pull request by Foxandxss**
Alright, based on what @robwormald did with clang, I formatted our filed.

Not that I am too fond of the style, but I guess it is for the best.

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
Only slightly related to this PR but could we have 120 columns instead of 100? It kills me how the ng2 code is formatted / wrapped....

And yeh, +1 for landing this ASAP as it is pretty much impossible to change formatting as soon as we start to have PRs

**Comment by Foxandxss** [comment]
SIGN ME IN!

No, seriously, I think that we the screens we have nowadays, 100 is too meh.

**Comment by Foxandxss** [comment]
I applied it here.

File: src/services/multiMap.ts [hunk 1]
```diff
@@ -1,30 +1,15 @@
...
 
-  entries() {
-    return Object.keys(this._map).map(key => {
-      return {
-        key: key,
-        value: this._map[key]
-      }
-    });
-  }
+  entries() { return Object.keys(this._map).map(key => {return {key: key, value: this._map[key]}}); }
```
Comment (wesleycho) [inline src/services/multiMap.ts]: It really makes this look a little harder to read :( .

Comment (Foxandxss) [inline src/services/multiMap.ts]: I agree.

**Comment by wesleycho** [comment]
LGTM - I can live with my little gripe

**Pull request closed without merging**